packages/shared/src/graphql/njord.ts

```diff
@@ -211,3 +211,10 @@ export const getTransactions = async ({
 
   return result.transactions;
 };
+
+export const awardMutationOptions = () => {
```

Comment:
Oh ok, this is pretty neat actually! :) 
Also promotes consistent re-use accross files.

Reply:
Yes, queryKey and queryFn is tied together and then you just shove them in whatever hook and you get types automatically.